content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+---
+title: "Contact Routing Guide"
+linkTitle: "Contact Routing Guide"
+description: "Set up a Sensu Enterprise Contact Routing."
+product: "Sensu Enterprise"
+version: "2.6"
+weight: 7
+menu:
+ sensu-enterprise-2.6:
+   parent: guides
+---
+
+- [Prerequisites](#prerequisites)
+- [Overview](#overview)
+  - [Contact Routing Basics](#contact-routing-basics)
+    - [Contact Configuration](#contact-configuration)
+    - [Check Configuration](#check-configuration)
+    - [Client Configuration](#client-configuration)
+- [Example Implementation](#example-implementation)
+  - [Default Configuration](#default-configuration)
+  - [Single Handler](#single-handler)
+      - [Single Handler with a Single Matching Contact](#single-handler-with-a-single-matching-contact)
+      - [Single Handler with a Single Non-Matching Contact](#single-handler-a-single-non-matching-contact)
```

Comment:
This link doesn't work. Not sure the fix here, might be a `-` being interpreted wrong.